app/javascript/packs/backup-code-analytics.js

```diff
@@ -0,0 +1,9 @@
+import { trackEvent } from '@18f/identity-analytics';
+
+const downloadLink = document.querySelector('a[download]');
+
+function trackDownload() {
+  trackEvent('IdV: download backup codes');
```

Comment:
Backup codes aren't really related to identity verification (the "IdV" prefix). Should we omit that or find an alternative prefix?

Reply:
Looking at [other events](https://github.com/18F/identity-idp/blob/main/app/services/analytics_events.rb), I see a handful of related events prefixed with "Multi-Factor Authentication:" . Would that fit here?